ui/src/app/images/views/ImageList/OtherImages/OtherImages.tsx

```diff
@@ -0,0 +1,118 @@
+import { useCallback } from "react";
+
+import { Strip } from "@canonical/react-components";
+import { usePrevious } from "@canonical/react-components/dist/hooks";
+import { useDispatch, useSelector } from "react-redux";
+import * as Yup from "yup";
+
+import OtherImagesSelect from "./OtherImagesSelect";
+
+import FormikForm from "app/base/components/FormikForm";
+import type { ImageValue } from "app/images/types";
+import { actions as bootResourceActions } from "app/store/bootresource";
+import bootResourceSelectors from "app/store/bootresource/selectors";
+import { BootResourceAction } from "app/store/bootresource/types";
+import {
+  splitImageName,
+  splitResourceName,
+} from "app/store/bootresource/utils";
+
+const OtherImagesSchema = Yup.object()
+  .shape({
+    images: Yup.array().of(
+      Yup.object().shape({
+        arch: Yup.string(),
+        os: Yup.string(),
+        release: Yup.string(),
+        subArch: Yup.string(),
+        title: Yup.string(),
+      })
+    ),
+  })
+  .defined();
+
+export type OtherImagesValues = {
+  images: ImageValue[];
+};
+
+const OtherImages = (): JSX.Element | null => {
+  const dispatch = useDispatch();
+  const otherImages = useSelector(bootResourceSelectors.otherImages);
+  const resources = useSelector(bootResourceSelectors.otherResources);
+  const saving = useSelector(bootResourceSelectors.savingOther);
+  const previousSaving = usePrevious(saving);
+  const eventErrors = useSelector(bootResourceSelectors.eventErrors);
+  const error = eventErrors.find(
```

Comment:
Eventually we probably should add a hook for getting the correct errors.

Reply:
For sure. I'm thinking maybe after all the images tables are built we can see what the common bits are